moninag/static/src/ts/app/checks/check-detail.component.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { ActivatedRoute, Params } from '@angular/router';
+import { Location } from '@angular/common';
+import 'rxjs/add/operator/switchMap';
+
+import { Check } from './check';
+import { ChecksService } from './checks.service';
+import { CheckUpdateComponent } from './check-update.component';
+
+
+@Component({
+    selector: 'checkdetail-app',
+    template: `
+    <div *ngIf="check">
+    <h6>Name:{{check.name}}</h6>
```

Comment:
use better something different than h6 for this section
a [definition list](https://www.w3schools.com/tags/tag_dl.asp)  for example or wrap it in a table